otwcode/otwarchive
# AO3-5008 Validate that subscribable exists before saving

## Issue

https://otwarchive.atlassian.net/browse/AO3-5008

## Testing

1. Create a work.
2. Login as another user and go to the work created in (1). Keep this page open.
3. Login (in an incognito browser or on another machine) as the user from (1) and delete the work you created.
4. Go to the page from (2) and click 'Subscribe'. You should see the error "Subscribable can't be blank".

You can do this with series and users as well.

## References

This might fix (one of) the causes of https://otwarchive.atlassian.net/browse/AO3-4728 and the Subscription page 500 errors.

Note: This doesn't validate the actual type of the subscribable object, so you could still (somehow) subscribe to something that's not a work, series, or user, but not to anything that doesn't exist in the database.

## Credit

cresenne (she/her)
